security/3.0/apidocs/doc-files/speclicense.html

```diff
@@ -0,0 +1,73 @@
+<html>
+<head>
+<title>Eclipse Foundation Specification License - v1.0</title>
+</head>
+<body>
+<h1>Eclipse Foundation Specification License - v1.0</h1>
+<p>By using and/or copying this document, or the Eclipse Foundation
+  document from which this statement is linked, you (the licensee) agree
+  that you have read, understood, and will comply with the following
+  terms and conditions:</p>
+
+<p>Permission to copy, and distribute the contents of this document, or
+  the Eclipse Foundation document from which this statement is linked, in
+  any medium for any purpose and without fee or royalty is hereby
+  granted, provided that you include the following on ALL copies of the
+  document, or portions thereof, that you use:</p>
+
+<ul>
+  <li> link or URL to the original Eclipse Foundation document.</li>
+  <li>All existing copyright notices, or if one does not exist, a notice
+      (hypertext is preferred, but a textual representation is permitted)
+      of the form: &quot;Copyright &copy; [$date-of-document]
+      Eclipse Foundation, Inc. <a href="https://www.eclipse.org/legal/efsl.php">
+      https://www.eclipse.org/legal/efsl.php</a>&quot;
+  </li>
+</ul>
+
+<p>Inclusion of the full text of this NOTICE must be provided. We
+  request that authorship attribution be provided in any software,
+  documents, or other items or products that you create pursuant to the
+  implementation of the contents of this document, or any portion
+  thereof.</p>
+
+<p>No right to create modifications or derivatives of Eclipse Foundation
+  documents is granted pursuant to this license, except anyone may
+  prepare and distribute derivative works and portions of this document
+  in software that implements the specification, in supporting materials
+  accompanying such software, and in documentation of such software,
+  PROVIDED that all such works include the notice below. HOWEVER, the
+  publication of derivative works of this document for use as a technical
+  specification is expressly prohibited.</p>
+
+<p>The notice is:</p>
+
+<p>&quot;Copyright &copy; 2018, 2020 Eclipse Foundation. This software or
+  document includes material copied from or derived from
+  <a href="https://jakarta.ee/specifications/security/2.0">
```

Comment:
Shouldn't this be updated in both HTML and PDF file so it references 3.0?

Reply:
I think this is the part that should be replaced by:

```html
<p>&quot;Copyright &copy; 2018 Eclipse Foundation. This software or
  document includes material copied from or derived from [title and URI
  of the Eclipse Foundation specification document].&quot;</p>
```

See e.g. https://github.com/jakartaee/servlet/blob/master/api/src/main/javadoc/doc-files/speclicense.html#L45

I'll update it, thx!

Reply:
Docs updated